examples/airline/configs/agents.py

```diff
@@ -0,0 +1,93 @@
+from configs.tools import *
+from data.routines.baggage.policies import *
+from data.routines.flight_modification.policies import *
+from data.routines.prompts import STARTER_PROMPT
+
+from openai_runner import Agent
+
+
+def transfer_to_flight_modification():
```

Comment:
Add comprehensive docstrings to the functions like `transfer_to_flight_modification` to explain their behavior, parameters, and return types. This will improve code readability and maintainability. [important]